bot/exts/evergreen/bookmark.py

```diff
@@ -103,8 +103,7 @@ async def bookmark(
             target_message = ctx.message.reference.resolved
 
         # Prevent users from bookmarking a message in a channel they don't have access to
-        permissions = ctx.author.permissions_in(target_message.channel)
-        if not permissions.read_messages:
+        if not target_message.channel.permissions_for(ctx.author).read_messages:
```

Comment:
This is a bit hard to read. Maybe it should stay split as before?

Reply:
Good point, i've updated both places to be the split way.